### PR TITLE
Hts 784 - Setup check eligibility page

### PR DESCRIPTION
**Pull request by thebailers**

### CODE REVIEW

File: app/uk.gov.hmrc.helptosavestridefrontend/controllers/SessionBehaviour.scala [hunk 1]
```diff
@@ -17,6 +17,7 @@
 package uk.gov.hmrc.helptosavestridefrontend.controllers
 
 import cats.instances.future._
+import controllers.Assets
```
Comment (AndyHWChung) [inline app/uk.gov.hmrc.helptosavestridefrontend/controllers/SessionBehaviour.scala]: This is probably not needed anymore

File: app/uk.gov.hmrc.helptosavestridefrontend/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -49,10 +50,10 @@
 
 @insideHeader = {
 @uiLayouts.header_nav(
-    navTitle = Some("help-to-save-stride"),
+    navTitle = Some("Check eligibility and create a Help to Save account"),
```
Comment (AndyHWChung) [inline app/uk.gov.hmrc.helptosavestridefrontend/views/govuk_wrapper.scala.html]: do we want this in the messages file?

File: app/uk.gov.hmrc.helptosavestridefrontend/views/helpers/input_text.scala.html [hunk 3]
```diff
@@ -21,11 +21,12 @@
 inputClass: Option[String] = None,
 hint: Option[String] = None,
 name: String = "value",
+containerClass: Option[String] = None,
 labelClass: Option[String] = None,
 errorMessage: Option[Html] = None
 )(implicit messages: Messages)
 
-<div class="form-group @if(errorMessage.isDefined){form-group-error}">
+<div class="form-group @if(errorMessage.isDefined){form-group-error} @if(containerClass.nonEmpty){@containerClass}">
```
Comment (AndyHWChung) [inline app/uk.gov.hmrc.helptosavestridefrontend/views/helpers/input_text.scala.html]: does the `containerClass` thing actually work here? It looks like you'll end up with `Some(blah blah)` in the resulting HTML. Doing something like `containerClass.getOrElse("")` seems to make more sense here

File: project/plugins.sbt [hunk 4]
```diff
@@ -19,3 +19,11 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0")
...
 
 addSbtPlugin("org.wartremover" % "sbt-wartremover" % "2.1.1")
+
+addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.2")
+
+addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.1.9")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "1.0.3")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.1")
```
Comment (AndyHWChung) [inline project/plugins.sbt]: what are these plugins used for?

Comment (thebailers) [reply project/plugins.sbt]: These are the plugins to enable us to write out own SASS and compile assets such as images which are specific to the service.

File: app/uk.gov.hmrc.helptosavestridefrontend/views/helpers/input_text.scala.html (hunk 3 above)
Comment (thebailers) [reply app/uk.gov.hmrc.helptosavestridefrontend/views/helpers/input_text.scala.html]: This does work for me, the intended class gets written to the element when provided.

**Pull request merged**